src/Uno.UI/UI/Xaml/Controls/Flyout/FlyoutPresenter.cs

```diff
@@ -10,8 +14,19 @@ public FlyoutPresenter()
 
 		protected override void OnPointerPressed(PointerRoutedEventArgs args)
 		{
-			// All pointer-related should be "eaten" to prevent closing
-			// the flyout when a tap is done in its content
+			if (this.GetTemplateRoot() is FrameworkElement root && this.Parent is FlyoutBasePopupPanel panel)
+			{
+				// allow flyout to be closed by clicking outside its content
+				var pointer = args.GetCurrentPoint(this);
+				var rootCoords = this.TransformToVisual(root).TransformPoint(pointer.Position);
+
+				if (0 > rootCoords.X || rootCoords.X > root.ActualWidth ||
+					0 > rootCoords.Y || rootCoords.Y > root.ActualHeight)
```

Comment:
Ask @dr1rrb, we probably have helpers for that.

Reply:
Not sure for this one ... maybe 🤷‍♂️